File: backend/components/metiers/administration_publique/guides/core/accessibility/guides/guide_accessibility.test.py
"""
Tests ultra avancés pour guide_accessibility.py (guides/core/accessibility/guides)
"""
from .guide_accessibility import get_accessibility_guide

def test_get_accessibility_guide():
    guide = get_accessibility_guide()
    assert isinstance(guide, dict)
    assert guide['title'] == 'Guide Accessibilité 3D'
    assert 'bestPractices' in guide
    assert 'integrationSteps' in guide
    assert 'Respecter les standards WCAG 2.1' in guide['bestPractices']

File: backend/components/metiers/administration_publique/guides/core/accessibility/guides/guide_accessibility.test.js
// guide_accessibility.test.js - Tests ultra avancés pour guide_accessibility.js (guides/core/accessibility/guides)
const { getAccessibilityGuide } = require('./guide_accessibility');
describe('guide_accessibility.js', () => {
  it('doit retourner un guide d’accessibilité complet', () => {
    const guide = getAccessibilityGuide();
    expect(guide.title).toBe('Guide Accessibilité 3D');
    expect(Array.isArray(guide.bestPractices)).toBe(true);
    expect(guide.bestPractices).toContain('Respecter les standards WCAG 2.1');
    expect(Array.isArray(guide.integrationSteps)).toBe(true);
    expect(guide.integrationSteps).toContain('Analyser les besoins utilisateurs');
  });
});
